react/features/app/actionTypes.js

```diff
@@ -21,3 +21,15 @@ export const APP_WILL_MOUNT = Symbol('APP_WILL_MOUNT');
  * }
  */
 export const APP_WILL_UNMOUNT = Symbol('APP_WILL_UNMOUNT');
+
+/**
+ * The type of the actions which signals that value for room url should be
+ * changed.
+ *
+ * {
+ *     type: SET_ROOM_URL,
+ *     url: string
```

Comment:
Sure, it will also help us to get rid of direct invocation of app's private method.